pages/clubs/ClubForm.jsx

```diff
@@ -60,7 +76,18 @@ var ClubForm = React.createClass({
               <h2>{ username ? this.state.headings[this.state.currentStep] : this.state.loginHeading }</h2>
             </Illustration>
           </section>
-          { this.state.loggedIn ? this.renderSteps() : this.renderLoginRequest() }
+
+          <div hidden={!this.state.loggedIn}>
+            <Form onUpdate={this.checkAnswer} fields={{studentData: studentData}}></Form>
+
+            <div hidden={!this.state.isInterestedStudent} className="navigation">
+              <a href="https://campus.mozilla.community/" target="_blank" className="button">Go to Mozilla Campus Club Page</a>
+            </div>
+          </div>
+
+          { this.state.isInterestedStudent === false && this.renderSteps() }
```

Comment:
This feels like abusing the logical operator to overcome a limitation in what you can write inside templating brackets, so it would probably be better to instead honour what JSX finds acceptable code in a templating block and move `this.state.isInterestedStudent === false` into the `renderSteps` function, so that it's clear what gets rendered:

```
renderSteps() {
  if (this.state.isInterestedStudent === false) {
    return null;
  }
  ...
}
```